listenbrainz/listen_replay/cli.py

```diff
@@ -21,4 +21,6 @@ def demo_user_replay(user_name):
         replayer.app.logger.error("Error while replaying listens: %s", str(e), exc_info=True)
         raise
 
-# add more commands here as needed
+@cli.command(name="find_bad_track_numbers")
+def find_bad_track_numbers():
+    print(TrackNumberUserFinder().find_users())
```

Comment:
Maybe consider writing in a format that could easily be used as input for another program? JSON?